agentlocal/agentlocal_test.go

```diff
@@ -0,0 +1,55 @@
+// pmm-admin
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package agentlocal provides facilities for accessing local pmm-agent API.
+package agentlocal
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestGetStatus(t *testing.T) {
+	type args struct {
+		requestNetworkInfo NetworkInfo
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *Status
+		wantErr bool
+	}{
+		{
+			name: "status",
+			args: args{
+				requestNetworkInfo: DoNotRequestNetworkInfo,
+			},
+			want:    nil,
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := GetStatus(tt.args.requestNetworkInfo)
+			if (err != nil) != tt.wantErr {
```

Comment:
we don't write test this way. Please use assert or require.|
If you want to write test for the call to pmm-agent it's better to mock the answer.

Reply:
Assert used in all cases. It was generated and I forgot that we use errors package for tests. Now I  think about this test, if I mock it then it actually unnecessary test I think. I added it, because codecov.io wanted to cover it. I will try invert test which make sense even with mock.

Reply:
@BupycHuk  I removed test and coverage is OK, even without it. So should I let as it is? Because I think with mock it is unnecessary test. Other issues should be fixed.